src/components/NewWorkflow/AddPhase/AddPhase.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import {connect} from 'react-redux';
+import AddTask from './AddTask';
+
+class AddPhase extends Component {
+
+    state={
+        displayCard: false,
+        phaseName: ''
```

Comment:
below on line 14 it looks like phaseName's data type is an object. I don't know if it's too important to reflect that here, instead of a string, but i feel like mismatching data types has wonked up my code in the past

Reply:
it is, thats a leftover from how i orginally did it, so when i call it i use this.state.phasname.name... but currently only using that to display, the call to redux is usinging different stuff.. why? because im a complicated person

Reply:
just kidding, sorry, why is because not finished